test/test_quantization.py

```diff
@@ -519,8 +519,32 @@ def test_quantized_rnn(self):
 
         torch.testing.assert_allclose(output_int8, ref_out)
         self.assertEqual(output_int8, ref_out)
-        for out, ref in zip(final_hiddens_int8, ref_hid):
-            torch.testing.assert_allclose(out, ref)
+        for out_val, ref_val in zip(final_hiddens_int8, ref_hid):
+            torch.testing.assert_allclose(out_val, ref_val)
+
+        class ScriptWrapper(torch.nn.Module):
+            def __init__(self, cell):
+                super(ScriptWrapper, self).__init__()
+                self.cell = cell
+
+            def forward(self, x, hiddens):
+                # type: (torch.Tensor, Tuple[torch.Tensor, torch.Tensor]) -> Tuple[torch.Tensor, Tuple[torch.Tensor, torch.Tensor]]
+                return self.cell(x, hiddens)
+
+        # TODO: TorchScript overloads don't work without this wrapper
+        cell_script = torch.jit.script(ScriptWrapper(cell_int8))
+        out_script, hid_script = cell_script(x, hiddens)
+        self.assertEqual(len(out_script), len(ref_out))
+        for out_val, ref_val in zip(out_script, ref_out):
+            torch.testing.assert_allclose(out_val, ref_val)
+
+        # Test save/load
+        b = io.BytesIO()
```

Comment:
nit: should we use the same tmpFile way (`with tempfile.TemporaryFile() as f:`) for testing?
https://github.com/pytorch/pytorch/blob/master/test/test_quantized_nn_mods.py#L133

I am OK if you think `io.BytesIO` is more convenient. Just want to make it consistent.

Reply:
No, I think we should move more of these toward using BytesIO. Using files in tests is generally considered bad practice because it introduces syscalls and dependencies on the execution environment, and thus can cause test flakiness/instability

Reply:
Make sense. I guess it would be great to change all `tempfile` to `io.BytesIO` then.

Reply:
Following this issue, creating a PR to fix the quantization related test: https://github.com/pytorch/pytorch/pull/25976